biomage/experiment/copy.py

```diff
@@ -2,9 +2,9 @@
 
 import click
 
-from utils.config import get_config
-from utils.constants import DEFAULT_SANDBOX, PRODUCTION, STAGING
-from utils.data import copy_experiments_to
+from ..utils.config import get_config
```

Comment:
why is this import change needed?

Reply:
Relative imports are far more reliable and should be used wherever possible. When I installed the package with pip, absolute imports didn't work.

Reply:
awesome, thanks for the explanation